logrotate.c

```diff
@@ -552,6 +553,7 @@ static int createOutputFile(char *fileName, int flags, struct stat *sb,
 		return -1;
 		}
 	}
+#endif
```

Comment:
The proposed change does not seem to be correct.  The condition of the if block you commented out is ```(!acl_set || force_mode)``` and ```force_mode``` can be true regardless of ```WITH_ACL```.